app/assets/javascripts/global_helpers.coffee

```diff
@@ -3,3 +3,5 @@ window.merge = (objects...) ->
   $.extend {}, objects...
 window.deepCopy = (originalObject, into = {}) ->
   $.extend(true, into, originalObject)
+window.fullDateTime = (dateTime) ->
```

Comment:
Je kunt in moment ook je eigen formats een naam geven. 
Mijn stategie is meestal om alle props als moment door te geven, zodat je er makkelijk mee kan werken. 
Scheelt een globale functie.